### ISSUE TITLE
Lag issue with bluespace jumped corpses

### ISSUE THREAD

**Issue by Massattack52** [shiptest-ss13/Shiptest#592]
Issue reported from  Round ID: [560](https://shiptest.ga/stats/rounds/560) (Shiptest)

Reporting client version: 514.1568

Heya, I'm Cronos Morphis in game, PromethiusX in Byond, and Massattack52 both here and in the Discord. I'd like to report a pretty straightforward issue. My corpse was brought onto a ship that subsequently blue space jumped out of existence. At the time, I was a ghost and outside of my body. Later on, out of curiosity, I attempted the 're-enter corpse' command, and something strange happened.

I suffered a huge lag spike that disconnected me from the server. Not so bad right? Wrong. Everyone else on the server reported severe lag around the same time. Chances are whatever processes attempted to place me back into a body that no longer existed had a hell of a time doing just that.

I was able to recreate this by re-entering my corpse two more times, just for safe measure.

Proposed Solutions: 

1. Make sure data for mobs is fully deleted during blue space jumps if this doesn't already occur.
2. Somehow make it so the re-enter corpse command doesn't work if the corpse was part of a blue space jump.

TLDR; Don't attempt to enter your bluespace jumped corpse, the server will have a bad time.
